graylog2-web-interface/src/components/event-notifications/event-notification-types/EmailNotificationSummary.jsx

```diff
@@ -21,16 +21,32 @@ class EmailNotificationSummary extends React.Component {
     return (
       <CommonNotificationSummary {...this.props}>
         <React.Fragment>
-          <dt>Sender</dt>
-          <dd>{notification.config.sender}</dd>
-          <dt>Subject</dt>
-          <dd>{notification.config.subject}</dd>
-          <dt>User recipients</dt>
-          <dd>{notification.config.user_recipients.join(', ') || 'No users will receive this notification.'}</dd>
-          <dt>Email recipients</dt>
-          <dd>{notification.config.email_recipients.join(', ') || 'No email addresses are configured to receive this notification.'}</dd>
-          <dt>Email Body</dt>
-          <dd><pre className={`${styles.bodyPreview} pre-scrollable`}>{notification.config.body_template}</pre></dd>
+          <tr>
+            <td>Sender</td>
+            <td>{notification.config.sender}</td>
+          </tr>
+          <tr>
+            <td>Subject</td>
+            <td>{notification.config.subject}</td>
+          </tr>
+          <tr>
+            <td>User recipients</td>
+            <td>{notification.config.user_recipients.join(', ') || 'No users will receive this notification.'}</td>
+          </tr>
+          <tr>
+            <td>Email recipients</td>
+            <td>
+              {notification.config.email_recipients.join(', ') || 'No email addresses are configured to receive this notification.'}
+            </td>
+          </tr>
+          <tr>
+            <td>Email Body</td>
+            <td>
+              <pre className={`${styles.bodyPreview} pre-scrollable`}>
+                {notification.config.body_template || <em>Empty body</em>}
```

Comment:
The email body looks a bit to wide. :smiley: 

![image](https://user-images.githubusercontent.com/461/61391150-5c74fe80-a8bc-11e9-9ddb-700d3985938b.png)

Reply:
✅